front/src/modules/timesStops/types.ts

```diff
@@ -10,6 +10,18 @@ export type TimeExtraDays = {
   dayDisplayed?: boolean;
 };
 
+export type TimesStopsRowNew = {
+  id: string;
```

Comment:
Can you also store the index of the row here ?
```suggestion
  id: string;
  index: number;
```